### ISSUE TITLE
gitea dump creates zip with backslashes as path separator

### ISSUE THREAD

**Issue by tobias--** [go-gitea/gitea#3585]
- Gitea version (or commit ref): 1.3.3
- Git version:
- Operating system: Windows 2012 R2
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description

When run on Windows the export/backup command `gitea.exe dump` creates a ZIP archive using backslashes as a path separator. This leads to a flat output on unzipping this archive on a unix system. 

Using unzip on unix:
```
~ % unzip -d gitea gitea-dump-1519632627.zip
Archive:  gitea-dump-1519632627.zip
   creating: gitea/custom/
   creating: gitea/custom\conf/
  inflating: gitea/custom\conf\app.ini
[...]
```

**Comment by tobias--** [comment]
Shouldn't this be already addressed in this commit? https://github.com/Unknwon/cae/commit/c6aac99ea2cae2ebaf23f26f76b04fe3fcfc9f8c

**Comment by lunny** [comment]
It seems a dependency's bug?

**Comment by tobias--** [comment]
I don't know. The commit I referenced should already handle backslashes. The vendor.json for 1.3.3 points to this exact commit (https://github.com/Unknwon/cae/commit/c6aac99ea2cae2ebaf23f26f76b04fe3fcfc9f8c, https://github.com/go-gitea/gitea/blob/7b297808cee599f5ff58b9f6afa64d3ee980998e/vendor/vendor.json#L24-L27)

gitea-repo.zip inside the dump also uses backslashes.

I worked around the issue by extracting the archive on a Windows machine and zipping it again.
